resources/helm/overlays/istio_cni/templates/daemonset-v2.4.yaml

```diff
@@ -67,7 +67,52 @@ spec:
             httpGet:
               path: /readyz
               port: 8000
-          command: ["install-cni"]
+          command:
+            - /bin/sh
+            - -c
+            - |
+              set -xe
+              # Collect host OS information
+              . /host/etc/os-release
```

Comment:
I don't have a RHEL instance to test with, but this doesn't work on fedora- should we guard against that? Previously, we also supported running the operator on kind IIRC, that won't work with this I guess